qubes_config/global_config.glade

```diff
@@ -1029,61 +1028,14 @@ Documentation will open in a disposable qube.</property>
                         </child>
-                        <child>
-                          <object class="GtkBox">
-                            <property name="visible">True</property>
-                            <property name="can-focus">False</property>
-                            <property name="halign">center</property>
-                            <property name="spacing">20</property>
-                            <child>
-                              <object class="GtkImage">
-                                <property name="visible">True</property>
-                                <property name="can-focus">False</property>
-                                <property name="halign">start</property>
-                                <property name="valign">center</property>
-                                <property name="pixel-size">48</property>
-                                <property name="icon-name">qubes-info</property>
-                              </object>
-                              <packing>
-                                <property name="expand">False</property>
-                                <property name="fill">True</property>
-                                <property name="position">0</property>
-                              </packing>
-                            </child>
-                            <child>
-                              <object class="GtkLabel" id="usb_info">
-                                <property name="visible">True</property>
-                                <property name="can-focus">True</property>
-                                <property name="halign">start</property>
-                                <property name="label" translatable="yes">To learn more about Qubes U2F Proxy service and its uses, you can check the &lt;a href="https://www.qubes-os.org/doc/u2f-proxy/"&gt;online documentation&lt;/a&gt;. 
-Documentation will open in a disposable qube.</property>
-                                <property name="use-markup">True</property>
-                                <property name="wrap">True</property>
-                              </object>
-                              <packing>
-                                <property name="expand">False</property>
-                                <property name="fill">True</property>
-                                <property name="position">1</property>
-                              </packing>
-                            </child>
-                            <style>
-                              <class name="info_box"/>
-                            </style>
-                          </object>
-                          <packing>
-                            <property name="expand">False</property>
-                            <property name="fill">False</property>
-                            <property name="position">10</property>
-                          </packing>
-                        </child>
```

Comment:
Moved the link into the text instead of having it in a separate box.

Reply:
For links in labels to be clickable, the label's id must be added to the `_handle_urls` function in global_config.py.

Reply:
Ah, okay, I'll make sure they're all added.